doc/userguide/support-status.rst

```diff
@@ -0,0 +1,490 @@
+Support Status
+==============
+
+Levels of Support
+-----------------
+
+Tier 1
+~~~~~~
+
+Core team develops and supports. Compiler errors or functional
+failures block git merges and releases. Functionality is enabled by
+default on the platforms it supports.
+
+Tier 2
+~~~~~~
+
+Core team develops and supports, sometimes with help from community
+members. Compiler errors or functional failures block git merges.
+Functionality problems such as ‘known issues’ or might still going
+into releases. Functionality may be disabled by default.
+
+Community
+~~~~~~~~~
+
+When a feature of Suricata is community supported, it means the
+OISF/Suricata development team won’t support it. This to avoid
```

Comment:
I would rephrase it do `won't directly support it` since we still might support it a bit